<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1040</width>
    <height>676</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1040</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="image1Open">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>515</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Image2Open">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>515</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image1Save">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>515</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Image2Save">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>515</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image1Black">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>515</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image1White">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>515</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>White</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image1GrayWA">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>515</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray weight avarege</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image1GrayCCIR">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray CCIR 601-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image2Black">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>515</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image2GrayWA">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>515</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray weight avarege</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image2GrayCCIR">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray CCIR 601-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image2White">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>515</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>White</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Y1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cb1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cr1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cb2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>540</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Y2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>540</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cr2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>540</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RGB1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RGB2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transform">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB-&gt;YCbCr-&gt;RGB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transform2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB-&gt;YCbCr-&gt;RGB</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>570</y>
      <width>671</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>JPEG</string>
     </attribute>
     <widget class="QPushButton" name="dctCr">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>DCT Cr</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dsCr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Downsampled Cr</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dsCb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Downsampled Cb</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dctY">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>DCT Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dctCb">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>DCT Cb</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Downsampling:</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="dsNone">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>None</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="ds2h1v">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>2h1v</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="ds1h2v">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>1h2v</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="ds2h2v">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>2h2v</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>421</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Quantization:</string>
      </property>
      <widget class="QRadioButton" name="quantMax">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>21</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>RadioButton</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="quantAlfaGamma">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>46</y>
         <width>21</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>RadioButton</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="quantStd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>72</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Std. matrix</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>33</x>
         <y>22</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ny:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>22</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Nc:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="Ny">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>18</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="Nc">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>18</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>48</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Yg:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>33</x>
         <y>48</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ya:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="gamma">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>46</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="alfa">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>46</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>193</x>
         <y>48</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ca:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>275</x>
         <y>48</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Cg:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="alfaC">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>46</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="gammaC">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>46</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Compress</string>
      </property>
     </widget>
     <widget class="QLabel" name="uncompressed">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Uncompressed: --</string>
      </property>
     </widget>
     <widget class="QLabel" name="compressed">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Compressed: --</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="quantStdHalf">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>98</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Std. matrix div by 2</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>PSNR</string>
     </attribute>
     <widget class="QPushButton" name="calcPSNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>calcPSNR</string>
      </property>
     </widget>
     <widget class="QLabel" name="PSNRLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PSNR: --</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
